src/commands/shadowGet/command.js

```diff
@@ -1,16 +1,25 @@
-import { validateElement, validateSelector } from '../../validators';
+import { validateSelector } from '../../validators';
+import resolveValue from '../../helpers/resolveValue';
 
-export default selector => {
+export default (selector, options) => {
   validateSelector(selector);
 
-  const element = Cypress.$(selector);
-  validateElement(element);
-
-  Cypress.log({
-    name: 'shadowGet',
-    message: `'${selector}'`,
-    consoleProps: () => ({ selector }),
+  Cypress._.defaults(options, {
+    log: true,
+    timeout: 10000,
```

Comment:
Is this setting default timeout of `10000` ms for all commands? Or just this one?